assets/src/components/routeLadders.tsx

```diff
@@ -15,6 +18,14 @@ const RouteLadders = ({
   timepointsByRouteId,
   selectedVehicleId,
 }: Props) => {
+  const { socket } = useContext(SocketContext)
+  useChannel<void | null>({
+    socket,
+    topic: "notifications",
+    event: "notification",
+    parser: console.log,
```

Comment:
Are we okay with logging on clients' consoles like this without a flag? I'm thinking yes it's fine, but want to double check.

Reply:
I think it would be OK, @ahakuta do you have any objection? Clients wouldn't see this unless they open the developer tools, and my thinking is that if you open that, you're probably fine with the occasional look behind the scenes.